tools/envoy_headersplit/replace_includes.py

```diff
@@ -0,0 +1,76 @@
+# !/usr/bin/env python3
```

Comment:
Please add tests for this script as well.

Reply:
The tests has been added